### PR TITLE
Visual Improvement: Blast and coin collect effects added to the game.

### PR DESCRIPTION
**Pull request by amrit-choudhary**
Blast and coin collect effects created using animating effects package.

ezgif-4-5fa8f747d588

### CODE REVIEW

File: src/game_screens/game_screen.py [hunk 1]
```diff
@@ -3,116 +3,172 @@
...
+                return Coin(self.res, self.size, self.surface, self.random_pos_x, -50)
+            elif coin_select_random < (self.params['silver_chance'] + self.params['gold_chance']):
+                return BlueCoin(self.res, self.size, self.surface, self.random_pos_x, -50)
+            else:
+                return Bomb(self.res, self.size, self.surface, self.random_pos_x, -50)
+
+        else:
+            return None
+
+    def new_random_x(self):
```
Comment (vineetjc) [inline src/game_screens/game_screen.py]: can name this better instead of `new_random_x()`

Comment (vineetjc) [reply src/game_screens/game_screen.py]: also, ~overlaping~ overlapping

File: src/managers/gameplay_parameters.py [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+        # Note: all chances in fraction. 0.10 = 10% chance of spawning
+
+        self.params_list['classic_easy'] = {
+            'key':              'classic_easy',
+            'display_name':     'Classic Mode',
+            'game_over_name':   'Classic Easy Mode',
+            'introduction':     ('The classic game mode, avoid bombs and '
+                                 'collect as many coins as possible in 30 seconds.'
+                                 ),
+            'game_mode':        Game_mode.GAME,
```
Comment (vineetjc) [inline src/managers/gameplay_parameters.py]: should we rename to `Game_mode.CLASSIC` for sticking to naming by game mode?

**Comment by vineetjc** [comment]
Merging this PR to accommodate significant changes, expecting required changes in future PRs.

File: src/game_screens/game_screen.py (hunk 1 above)
Comment (amrit-choudhary) [reply src/game_screens/game_screen.py]: Fixed both

File: src/managers/gameplay_parameters.py (hunk 2 above)
Comment (amrit-choudhary) [reply src/managers/gameplay_parameters.py]: Fixed it. changed .GAME to .CLASSIC and also changed the game_screen to classic_game_screen

**Comment by amrit-choudhary** [comment]
Can't replicate the highscore score, will need more info on that

1v1:
split screen is difficult to implement.
if they overlap at the point where coin collides, player 1 will get the coin 
we can add some collision system later
when player 1 dies, it will show player 2 won. winner's points is the highscore

Infinite:
score at end = score+time survived

Hardcore:
score = time * some factor

AI is done

Heist:
you loose if you miss X coins continuously. I'm still testing what should be X.

Collision from sides is a bit complex, add that as an issue, i can do that later.

**Pull request merged**